### ISSUE TITLE
Accessory state not updating after Homebridge restart.

### ISSUE THREAD

**Issue by damianmroz** [arachnetech/homebridge-mqttthing#227]
So i updated my config json and restarted homebridge.
I noticed that accessories on home app do not update the state.
Even when i force close and reopen home app accessories are not updating the state.
Light can be ON and in Home App it is OFF.
This same goes for dimmer, values it always showing is 0% even when lamp is set to 100%.
I can control lights it starts refreshing state when i switch off and then on light then it is updated in home app.
Is this a feature or bug?
My config:
```
{
    "password": "XXX",
    "accessory": "mqttthing",
    "name": "Table Lamp",
    "username": "XXX",
    "type": "lightbulb",
    "url": "mqtt://192.168.1.30",
    "offValue": "OFF",
    "topics": {
        "setBrightness": "cmnd/tablelamp/dimmer",
        "setOn": "cmnd/tablelamp/power",
        "getOn": "stat/tablelamp/POWER",
        "setColorTemperature": "cmnd/tablelamp/CT",
        "getColorTemperature": {
            "topic": "stat/tablelamp/RESULT",
            "apply": "return JSON.parse(message).CT;"
        },
        "getBrightness": {
            "topic": "stat/tablelamp/RESULT",
            "apply": "return JSON.parse(message).Dimmer;"
        }
    },
    "onValue": "ON"
}
```

**Comment by arachnetech** [comment]
Currently, homebridge-mqttthing makes no attempt to store accessory state across homebridge restarts. If you want to reset state to known values on a restart, you can use the `startPub` option to publish ‘get’ and ‘set’ values to update HomeKit and your accessory.

I’m building a list of requested features which I’ll add this to.

**Comment by damianmroz** [comment]
Ok. Is there any way to make StartPub working in Hoobs? It is not accepting my config with this parameter.

**Comment by arachnetech** [comment]
That’s irritating. I changed the configuration format of startPub so that it could be handled by `config-ui-x`, but it’s possible that it still won’t work with HOOBS. Does HOOBS allow the JSON configuration to be edited directly?

**Comment by damianmroz** [comment]
Yes I can configure JSON directly but it is not accepting config with StartPub.

**Comment by arachnetech** [comment]
`startPub` works in HOOBS if configured directly in the XML ('Advanced'  configuration). I just tested with this accessory in latest HOOBS and the topics were correctly published on start-up (making the initial state of my light 'on'):

```
        {
            "accessory": "mqttthing",
            "plugin_map": {
                "plugin_name": "homebridge-mqttthing",
                "index": 0
            },
            "name": "My light bulb",
            "type": "lightbulb",
            "onValue": "true",
            "offValue": "false",
            "topics": {
                "getOn": "test/get/on",
                "setOn": "test/set/on"
            },
            "startPub": [
                {
                    "topic": "test/get/on",
                    "message": "true"
                },
                {
                    "topic": "test/set/on",
                    "message": "true"
                }
            ],
            "url": "mqtt://192.168.10.35"
        }
```

Configuration through the HOOBS configuration GUI is currently horrible (if not unusable) because HOOBS doesn't yet support all settings available in *config-ui-x*. However, I believe this is being addressed in the next HOOBS release (version 3.2). For now, I'd strongly recommend editing the XML directly.

**Comment by ramirocaballero** [comment]
HI arachnetech! Is it possible to get the Brightness after restart homebridge?
